services/kommander-flux/2.0.1/templates/v1_resourcequota_critical-pods-kommander-flux.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
```

Comment:
TODO: figure out if this is needed or can be removed (cross reference the quota added for gke fix).

If not, we can drop this file in the `flux-update.sh` script.

Reply:
let's get rid of our resourcequota creation that we added to the flux controller https://github.com/mesosphere/kommander/blob/main/federation/pkg/controllers/kommanderclusters/kommandercluster_flux_controller.go#L198 I'd rather just stick with upstream than deploying our own (which we did purely to get around the GKE issue). we can take care of that after this bump though, we don't need to block this flux bump on that --> filed https://d2iq.atlassian.net/browse/D2IQ-98689 to take care of this separately